.github/workflows/go-test.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "ubuntu", "windows", "macos" ]
+        os: [ "ubuntu-22.04", "ubuntu-latest", "windows-latest", "macos-latest" ]
```

Comment:
Any changes to this file will get overwritten the next time https://github.com/protocol/.github is pushed to.

Off the top of my head, the easiest way around it would be to add a new workflow in this repo only which runs `go test` on `ubuntu-22.04`.

Reply:
Oh, thank you, I understand the problem. Then I think it's better to wait  https://github.com/protocol/.github/issues/349 .

Reply:
I think it's done.